README.md

```diff
@@ -61,26 +61,16 @@ pip install enzyme-jax
 Requirements: `bazel-5.3.0`, `clang++`, `python`, `python-virtualenv`,
 `python3-dev`.
 
+Build our extension with:
 ```sh
-# Get submodules.
-git submodule update --init --recursive
-
-# [Optional] Build and install JAX into a new virtual environment.
-# Refer to https://jax.readthedocs.io/en/latest/developer.html for more details.
-virtualenv .venv
-pip install numpy wheel
-cd jax
-python build/build.py
-pip install dist/*.whl --force-reinstall
-pip install -e .
-cd ..
-
-# Build our extension.
 # Will create a whl in bazel-bin/enzyme_jax-VERSION-SYSTEM.whl
 bazel build :enzyme_jax
 ```
 
-After changing LLVM, it is necessary to rebuild and reinstall JAX.
+Finally, install the built library with:
+```sh
+pip install bazel-bin/enzyme_jax-VERSION-SYSTEM.whl
```

Comment:
This is repeated three lines down, maybe remove the latter one? Also maybe give a warning thata you cannot run code using it, in the root of the git directory (and thus you need to cd into test)

Reply:
I did it. I resolved the comment. I wrote the documentation. YAY!